storage/innobase/include/arch0arch.h

```diff
@@ -1396,6 +1403,8 @@ class Arch_Log_Sys {
   mutexes. Same is true for #m_archived_lsn. */
   Arch_State m_state;
 
+  lsn_t m_stop_lsn{LSN_MAX};
```

Comment:
This variable is accessed by multiple threads without proper synchronization.

Reply:
Fixed and squashed in https://github.com/facebook/mysql-5.6/pull/1250/commits/51ae4a8539e8b79e0f2a8f3ea146962ca80424c8